app/models/API1811/FinancialRequestHIP.scala

```diff
@@ -21,26 +21,26 @@ import play.api.libs.json.{Json, OWrites}
 case class FinancialRequestHIP(
                                 taxRegime: String,
                                 taxpayerInformation: TaxpayerInformation,
-                                targetedSearch: TargetedSearch,
-                                selectionCriteria: SelectionCriteria,
-                                dataEnrichment: DataEnrichment
+                                targetedSearch: Option[TargetedSearch],
+                                selectionCriteria: Option[SelectionCriteria],
+                                dataEnrichment: Option[DataEnrichment]
                               )
 
 case class TaxpayerInformation(idType: String, idNumber: String)
-case class TargetedSearch(searchType: Option[String], searchItem: Option[String])
+case class TargetedSearch(searchType: String, searchItem: String)
 case class SelectionCriteria(
-                              dateRange: Option[DateRange],
-                              includeClearedItems: Option[Boolean],
-                              includeStatisticalItems: Option[Boolean],
-                              includePaymentOnAccount: Option[Boolean]
+                              dateRange: DateRange,
```

Comment:
DateRange is Optional.
DateRange contains 3 mandatory params including dateType
<img width="469" height="592" alt="image" src="https://github.com/user-attachments/assets/0abf2e58-e0c7-4cae-bfaa-8a374af4d800" />